test/puppetlabs/dujour/version_check_test.clj

```diff
@@ -19,14 +21,22 @@
            (get-coords {:group-id "foo.foo"
                         :artifact-id "foo"})))))
 
+(defn parse-params [params encoding]
+      (let [params (codec/form-decode params (str encoding))]
+           (if (map? params) params {})))
+
 (defn update-available-app
-  [_]
-  {:status 200
-   :body (json/generate-string {:newer true
-                                :link "http://foo.com"
-                                :message "Howdy!"
-                                :product "foo"
-                                :version "9000.0.0"})})
+  [req]
+  (let [params (ring.util.codec/form-decode (ring.util.codec/percent-decode (:query-string req)))
+        certname (params "certname")
+        certname (str certname)
+        certname (.replace certname "\\" "")]
```

Comment:
Is `str` required in this case? Also I think this can be simplified to just read as:

```clojure
certname (-> (params "certname")
             str ;this might be required?
             (.replace "\\" ""))
```